students/dave_arasim/session06/cigar_party.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
```

Comment:
Someone else's file. Same for all the remaining files. Not sure how this happened, but we'll need to remove all these files (by others) in your PR before merging it. I'll think about how to do that offline...